src/createContext.ts

```diff
@@ -51,44 +53,50 @@ export const defineSymbol = (context: Context, name: string, value: Value) => {
   })
 }
 
+export const defineBuiltin = (context: Context, name: string, f: Function) => {
+  let wrapped = (...args: any) => f(...args)
+  wrapped.toString = () => {
+    const params = getParameterNames(f).join(', ')
```

Comment:
Recomputing the string representation each time toString is called gives the impression that the function might change signature when reading the code. It should probably be computed outside of the function.